tensorlayer/prepro.py

```diff
@@ -3270,3 +3278,529 @@ def sequences_get_mask(sequences, pad_val=0):
             else:
                 break  # <-- exit the for loop, prepcess next sequence
     return mask
+def crop_meta_image(image,annos,mask):
```

Comment:
Can’t understand the meaning of this API, it is about pose?  The comments of arguments  are not clear